src/commands/synchronize.ts

```diff
@@ -1,18 +1,18 @@
 import { Command, flags } from '@oclif/command';
 
-import { log } from './logger';
-import { handleFlags } from './cli';
-import { getElapsedTime } from './util/ts';
-import SyncResult from './domain/SyncResult';
-import SyncParams from './domain/SyncParams';
-import { printError, printLine } from './util/io';
-import ExecutionContext from './domain/ExecutionContext';
-import { loadConfig, resolveConnections } from './config';
+import { log } from '../logger';
+import { handleFlags } from '../cli';
+import { getElapsedTime } from '../util/ts';
+import SyncResult from '../domain/SyncResult';
+import SyncParams from '../domain/SyncParams';
+import { printError, printLine } from '../util/io';
+import ExecutionContext from '../domain/ExecutionContext';
+import { loadConfig, resolveConnections } from '../config';
 
 /**
- * SyncDB CLI handler.
+ * Synchronize command handler.
  */
-class SyncDb extends Command {
+class Synchronize extends Command {
   static description = 'Synchronize database';
 
   /**
```

Comment:
Perhaps `generate-connections` can be a command now instead of a flag for `synchronize`.

I think this looks a bit weird:
```bash
yarn sync-db synchronize --generate-conections
```

This keeps it simple:
```bash
yarn sync-db generate-connections
```

Reply:
We'll remove generate-connections now. It's not useful as the env vars being used by this flag can directly be used without a connections file if given. 

So, it would be simpler for the users to know less on this. 

Thanks for pointing it out thought, I'll put a deprecation log for it.